account_move_template/wizard/select_template.py

```diff
@@ -0,0 +1,111 @@
+# Copyright 2015-2017 See manifest
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+import time
+from odoo import models, fields, api
+
+
+class WizardSelectMoveTemplate(models.TransientModel):
+    _name = "wizard.select.move.template"
+
+    template_id = fields.Many2one('account.move.template', required=True)
+    partner_id = fields.Many2one('res.partner', 'Partner')
```

Comment:
A selectable date field for posting plus bulk creation for multiple templates would make things easier